bork/cli.py

```diff
@@ -67,9 +70,25 @@ def main():
     if verbose:
         sys.argv.remove('--verbose')
 
+    try:
+        import coloredlogs
```

Comment:
This should solve the linting problem:

```suggestion
        import coloredlogs  # pylint: disable=import-outside-toplevel
```